pom.xml

```diff
@@ -26,6 +26,7 @@
         <logstash.version>7.0.1</logstash.version>
         <springdoc.version>1.6.5</springdoc.version>
         <testcontainers.version>1.16.3</testcontainers.version>
+        <spring.boot.version>2.6.3</spring.boot.version>
```

Comment:
Tror vi skal ta bort denne, fordi versjoner implisitt er satt gjennom parent spring-boot-starter-parent, som da sørger for at dependencies er valgt og testa sammen for å passe til satt versjon av Spring Boot (gjennom parent). Best practice å unngå eksplisitte versjoner med mindre man har en god grunn (eks. log4j-problemet nylig).